app/scripts/modules/kubernetes/serverGroup/details/details.html

```diff
@@ -74,6 +74,8 @@ <h4 class="text-center" ng-if="serverGroup.isDisabled">[SERVER GROUP IS DISABLED
         <dd ng-show="serverGroup.revision">{{serverGroup.revision}}</dd>
         <dt>Kube UI</dt>
         <dd><a href={{ctrl.uiLink()}} target="_blank">{{serverGroup.name}}</a></dd>
+        <dt>TERM Period</dt>
```

Comment:
This doesn't feel like a top-level detail - could you find somewhere else on this panel for this value? (It could also live on the instance details panel).